examples/neural_networks/plot_random_neural_network.py

```diff
@@ -0,0 +1,155 @@
+"""
+========================================================================
+Impact of varying hyperparameters alpha and weight_scale on toy datasets
+========================================================================
+
+This illustrates how the random neural network behaves when varying the 
+regularization terms, weight_scale and alpha. Regularization usually
+affects the regularity or smoothness of the decision function.
+
+This example generates two plots, corresponding to each of the hyperparameters.
+They control regularization in that they constrain the coefficients of the
+decision function.
+
+If there is high bias - as can be indicated by a high training
+error, decreasing alpha or increasing weight_scale would decrease bias and 
```

Comment:
open dash is closed by comma here. (Also if you decide to stick with dashes, two of them would be more typographical